frontend/src/config.ts

```diff
@@ -1,4 +1,5 @@
-export const BACKEND_URL = 'https://backend.aadesh-kulkarni08.workers.dev';
+// export const BACKEND_URL = 'https://backend.aadesh-kulkarni08.workers.dev';
+export const BACKEND_URL = 'http://localhost:39013';
```

Comment:
The BACKEND_URL should be pointing to workers.dev, not localhost. Please rollback changes to this file.